_posts/2020-12-21-moveit-for-ros-2-migration-update.md

```diff
@@ -0,0 +1,21 @@
+---
+date: 2020-12-21 09:48:22 -0700
+layout: post
+author: Henning Kayser
+comments: false
+title: MoveIt for ROS 2 Migration Update
+media_type: image
+media_link: "/assets/images/blog_posts/2020-12-21-ros-2-tsc-moveit-2-update.png"
+description: Progress update and learnings about the MoveIt 2 Migration progress as
+  shared at the TSC in December 2020.
+categories:
+- MoveIt
+- ROS 2
+- TSC
+
+---
+With the end of the year just right around the corner we’ve taken a look back on the progress and challenges of the ROS 2 migration of MoveIt. Checkout our [latest presentation on MoveIt 2](https://docs.google.com/presentation/d/1pcW9v-ynGKuSqNtY-QJ4Ts5SkRZS6DmRARFYPisIqSU/edit?usp=sharing).
```

Comment:
With the end of the year just right around the corner -->  With the end of the year just around the corner